resources/check.rb

```diff
@@ -30,9 +30,9 @@
 provides :sensu_check
 
 property :check_hooks, Array
-property :command, String, default: '/bin/true', required: true
+property :command, String, default: '/bin/true'
```

Comment:
I think these might want to remain required :thinking: whats the rationale for not specifying a command?

Reply:
Cookstyle is unhappy if a property has both a default, and is required: https://www.rubydoc.info/gems/cookstyle/RuboCop/Cop/Chef/ChefRedundantCode/PropertyWithRequiredAndDefault

I don't know why this is has a default command, but for fixing the cookstyle, I opted for just removing the part that would break the least. I doubt there's someone out there using /bin/true as the check however, and I agree that this probably should be required.

Reply:
Ya I prefer to remove the default in these (check and subscriptions) cases then, handlers is a bit different I guess because of the changes. I do need to check that there is not a valid use case for empty subscriptions, I am not sure how "unpublished checks" work in sensu-go (or if they are still a thing) but that might be one use case we need to double check.

Reply:
Verified according to the check spec that subscriptions are command and subscriptions are required.